pay-api/src/pay_api/models/fee_schedule.py

```diff
@@ -189,3 +228,13 @@ class Meta:  # pylint: disable=too-few-public-methods
         ],
     )
     filing_type = ma.Nested(FilingTypeSchema, many=False, data_key="filing_type_code")
+
+class FeeDetailsSchema(Schema):
```

Comment:
We're trying to move away from using `marshmallow / flask-marshmallow` if you look at all the most recent code it's using `cattrs`